docs/docs/manual/starting.md

```diff
@@ -1,33 +1,23 @@
 ---
-
 id: starting
-
 title: Starting a project
-
 sidebar_label: Starting a project
-
 ---
 
-
 ## Overview
 
 OpenRefine doesn’t allow you to create a dataset from nothing - you can only start a project by importing in some existing data.
```

Comment:
Nitpicking: I would intuitively start by describing what OpenRefine allows you to do, rather than what it does not (to avoid the slight frustration)